pkg/ddl/ddl.go

```diff
@@ -560,26 +561,49 @@ func (d *ddl) RegisterStatsHandle(h *handle.Handle) {
 
 // asyncNotifyEvent will notify the ddl event to outside world, say statistic handle. When the channel is full, we may
 // give up notify and log it.
-func asyncNotifyEvent(jobCtx *jobContext, e *notifier.SchemaChangeEvent, job *model.Job) {
+func asyncNotifyEvent(jobCtx *jobContext, e *notifier.SchemaChangeEvent, job *model.Job, sctx *sess.Session) error {
+	if intest.InTest && notifier.DefaultStore != nil {
```

Comment:
we can move this if branch after line 592. So checking system database is not duplicated